### PR TITLE
Add HALBORN Audit Report

### PR DESCRIPTION
**Pull request by hellwolf**

### CODE REVIEW

File: packages/ethereum-contracts/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    - new functions in IConstantFlowAgreementV1
+  - Other changes:
+    - emit FlowUpdatedExtension event
+
+### Fixes
+
+- Fix ERC1820 artifact issue (#759)
+- Fix an issue with output dir (#748)
+- [SECURITY] InstantDistributionAgreement.claim should be guarded by ctx check.
+- Inteface renaming fix re patrician periods (#747)
```
Comment (0xdavinchee) [inline packages/ethereum-contracts/CHANGELOG.md]: Interface*

File: packages/ethereum-contracts/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+### Changed
+
+- Disable zero address subscription for IDA (#751)
+- Update dev dependencies (#694)
+- Use hardhat for ethereum-contracts tests (#727)
+- Update to solc 0.8.13 (#721)
+
+### Improvements
+
+- Redeuce trust assumptions host <-> agreements (#749)
```
Comment (0xdavinchee) [inline packages/ethereum-contracts/CHANGELOG.md]: *Reduce

File: packages/ethereum-contracts/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+- Additional IDA test cases (#722)
+
+## [v1.1.1] - 2022-03-18
+
+### Fixes
+
+- Add Strings.sol back to repository (#715)
```
Comment (0xdavinchee) [inline packages/ethereum-contracts/CHANGELOG.md:42]: I thought we ended up removing this though?

Comment (hellwolf) [reply packages/ethereum-contracts/CHANGELOG.md:42]: yea, that was 1.1.1

File: packages/ethereum-contracts/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+- Add SuperApp factory registration script (#657)
+- CFALibraryV1: added flow() and flowWithCtx() functions (#595)
+- Self approve transfer (#598)
+- External set/clear config added to governance (#624)
+
+### Fixes
+
+- Add missing event emits (#656)
+- CFALibraryV1: Change solidity pragma in cfa lib to support versions 0.8 & later (#642)
+- Multiple scripts fixes re: do not engage with unitialized super token proxies (#603)
```
Comment (0xdavinchee) [inline packages/ethereum-contracts/CHANGELOG.md:88]: uninitialized*

File: packages/ethereum-contracts/CHANGELOG.md (hunk 3 above)
Comment (hellwolf) [reply packages/ethereum-contracts/CHANGELOG.md:42]: I will added a note in new release that it's remove dagain

File: packages/ethereum-contracts/CHANGELOG.md (hunk 1 above)
Comment (hellwolf) [reply packages/ethereum-contracts/CHANGELOG.md]: fixed

File: packages/ethereum-contracts/CHANGELOG.md (hunk 2 above)
Comment (hellwolf) [reply packages/ethereum-contracts/CHANGELOG.md]: fixed

File: packages/ethereum-contracts/CHANGELOG.md (hunk 3 above)
Comment (0xdavinchee) [reply packages/ethereum-contracts/CHANGELOG.md:42]: oh that makes sense

**Review by 0xdavinchee — APPROVED** [review]
looks good, LGTM!

**Comment by hellwolf** [comment]
https://xkcd.com/759/

image

**Pull request merged**